client/src/pages/HomePage.js

```diff
@@ -1,56 +1,89 @@
-import React, { useContext, useEffect, useState } from "react";
-import AppBarLoggedIn from "../layout/AppBarLoggedIn";
-import { Grid, Typography, Box } from "@material-ui/core";
-import { UserContext } from "../context/user";
-import Mention from "../layout/Mention";
-import MentionList from "../layout/MentionList";
-import { makeStyles } from "@material-ui/core/styles";
-import Spinner from "../layout/Spinner";
-import { getMentions } from "../hooks/getMentions";
+import React, { useContext, useEffect, useState } from 'react';
+import AppBarLoggedIn from '../layout/AppBarLoggedIn';
+import { Grid, Typography, Box } from '@material-ui/core';
+import { UserContext } from '../context/user';
+import Mention from '../layout/Mention';
+import MentionList from '../layout/MentionList';
+import { makeStyles } from '@material-ui/core/styles';
+import Spinner from '../layout/Spinner';
+import { getMentions } from '../hooks/getMentions';
+import InfiniteScroll from 'react-infinite-scroller';
+import axios from 'axios';
```

Comment:
Can you confirm prettier settings with your team? I think you have single quotes and others have double quotes set.

Reply:
I switched back to double quotes, but i'll check with the team later today